bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/entity/Street.kt

```diff
@@ -51,4 +51,23 @@ class Street(
     @Column
     val additionalNameSuffix: String? = null,
 
-    )
\ No newline at end of file
+    ) {
+
+    override fun equals(other: Any?): Boolean {
```

Comment:
Was this method overlooked? I think the default equals for data classes can be used for the Street as well, correct?

Reply:
And `Street` is not a data class yet.